draft-irtf-cfrg-frost.md

```diff
@@ -252,11 +253,13 @@ group as the object `G` that additionally defines helper functions described bel
 for `G` is addition `+` with identity element `I`. For any elements `A` and `B` of the group `G`,
 `A + B = B + A` is also a member of `G`. Also, for any `A` in `G`, there exists an element
 `-A` such that `A + (-A) = (-A) + A = I`. For convenience, we use `-` to denote
-subtraction, e.g., `A - B = A + (-B)`. Scalar multiplication is equivalent to the repeated
+subtraction, e.g., `A - B = A + (-B)`. Integers, taken modulo the group order `p`, are called
+scalars; arithmetic operations on scalars are implicitly performed modulo `p`. Since `p` is prime,
+scalars form a finite field. Scalar multiplication is equivalent to the repeated
 application of the group operation on an element `A` with itself `r-1` times, denoted as
-`ScalarBaseMult(A, r)`. We denote the sum, difference, and product of two scalars using the `+`, `-`,
+`ScalarMult(A, r)`. We denote the sum, difference, and product of two scalars using the `+`, `-`,
```

Comment:
I see the API changed here but no changes throughout the document or in the implementation, so reminder here to check everywhere to make sure these changes carry through.